.github/workflows/publish.yml

```diff
@@ -12,21 +12,39 @@ jobs:
     name: Publish
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/setup-go@v2
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Check Release
+      id: check-release
+      run: |
+        tag="$(git describe --tags --exact-match 2> /dev/null || :)"
+        if [[ -z "$tag" ]];
+          then
+            echo "::set-output name=IS_RELEASE::false"
+          else
+            echo "::set-output name=IS_RELEASE::true"
+        fi
```

Comment:
This script is an excerpt from the existing publish script here:

https://github.com/vercel/vercel/blob/ecb44812263a03803157ad258c2f0f3cd0fc83ef/utils/publish.sh#L18-L23